pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -129,7 +129,11 @@ let
 
     strictDeps = true;
 
-    env = lib.optionalAttrs (stdenv.isDarwin && stdenv.isx86_64) {
+    env = {
+      # Tell ninja to avoid ANSI sequences, otherwise we don’t see build
+      # progress in Nix logs.
+      NINJA = "TERM=dumb ninja";
```

Comment:
Not setting `TERM=dumb` globally because Node.js v18 needs https://github.com/nodejs/node/commit/f76b28f2cf0cde9dd9d4476b9bbd772bc48f722c backport.

Reply:
We should write that into a comment

Reply:
Hmm, this is unfortunate. I think Nix build environments have enough of a PTY to run these tests, but we’re using `TERM=dumb` to get Ninja to give us more detailed output. It’d be nice if we could still run those tests, but it’s probably not that important.

Reply:
These tests run fine _unless_ `TERM=dumb` is set 😅

Reply:
Yeah. We could override `TERM` for just those tests, but it’d be ugly. I’m guessing Ninja has probably already been approached about an alternate way to toggle non‐fancy build output and maybe rejected it. Oh well.

Reply:
Filed https://github.com/ninja-build/ninja/issues/2475.